rolling/distribution.yaml

```diff
@@ -7202,6 +7202,22 @@ repositories:
       url: https://github.com/PickNikRobotics/rviz_visual_tools.git
       version: ros2
     status: maintained
+  sbg_driver:
+    doc:
+      type: git
+      url: https://github.com/SBG-Systems/sbg_ros2.git
+      version: master
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/SBG-Systems/sbg_ros2-release.git
```

Comment:
For Rolling, all release repositories must be in https://github.com/ros2-gbp/ .  To move your release repository over there, please see the documentation at https://github.com/ros2-gbp/ros2-gbp-github-org/blob/latest/README.md